onboarding/tests/test_array.py

```diff
@@ -1 +1,9 @@
-# write some tests here!
+import pytest
+from ..array_1 import find_min_index
```

Comment:
This should be array not array_1. That's causing the tests to fail. Before submitting check with `pytest` if your code is working fine :)